### PR TITLE
Update EXP Yield for Indigo Disk mons + other data fixes

### LINKED ISSUE #3882: The Indigo Disk mons are missing exp yield
### Description

title

### Version

upcoming (Edge)

### Upcoming Version

_No response_

### Discord contact info

_No response_

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Finally found a resource that had accurate exp yields. Also fixed some straggling incorrect data.
From here: https://docs.google.com/spreadsheets/d/1MAQLLg60Y036EQYHtz0PjCBLEYcLgy6mhjsaUDxiSFY/edit?usp=sharing

## Issue(s) that this PR fixes
Fixes #3882 

## **Discord contact info**
kittenchilly

### CODE REVIEW

File: src/data/pokemon/species_info/gen_1.h [hunk 1]
```diff
@@ -10220,7 +10220,7 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
         .types = { TYPE_FIGHTING, TYPE_FIGHTING },
         .evYield_Attack = 2,
         .abilities = { ABILITY_INTIMIDATE, ABILITY_ANGER_POINT, ABILITY_CUD_CHEW },
-        .bodyColor = BODY_COLOR_BROWN,
+        .bodyColor = BODY_COLOR_BLACK,
```
Comment (Bassoonian) [inline src/data/pokemon/species_info/gen_1.h:10223]: I'd agree that this is probably out of scope (just lmk if you don't want to do it), but all the traits shared between Paldean Tauros forms should probably be inside the base stats macro (which should be renamed appropriately in that case)

Comment (AsparagusEduardo) [reply src/data/pokemon/species_info/gen_1.h:10223]: it was handled in #3971, I'll merge with upcoming.

**Comment by AsparagusEduardo** [comment]
These should be moved to master, since it's a fix to incorrect data

**Comment by Bassoonian** [comment]
> These should be moved to master, since it's a fix to incorrect data

The Indigo Disk mons aren't yet in master, though. I guess that means that maybe the PR should be split?

**Comment by kittenchilly** [comment]
> These should be moved to master, since it's a fix to incorrect data

Indigo disk mons are only in upcoming.

**Comment by AsparagusEduardo** [comment]
> The Indigo Disk mons aren't yet in master, though. I guess that means that maybe the PR should be split?

Ye

**Comment by kittenchilly** [comment]
Actually, I think edu's pr to master did do most of the colors i changed. Didn't realize.
I'll revert that commit, although there are still some but I guess that can be a separate pr

**Comment by kittenchilly** [comment]
> Actually, I think edu's pr to master did do most of the colors i changed. Didn't realize. I'll revert that commit, although there are still some but I guess that can be a separate pr

In particular, I dont think edu's pr covers dialga, and morpeko, or the hisuian mons + evos

**Review by AlexOn1ine — APPROVED** [review]

**Pull request merged**